src/Microsoft.DotNet.ProjectModel/Compilation/LibraryAsset.cs

```diff
@@ -17,6 +17,7 @@ public struct LibraryAsset
         public string ResolvedPath { get; }
         public string FileName => Path.GetFileName(RelativePath);
         public Action<Stream, Stream> Transform { get; set; }
+        public bool IsPlaceholder => FileName.Equals("_._");
```

Comment:
Actually, I think we're going to need them in the exporter. We need to call out the difference between no assets were **compatible** and no assets are **needed** and preserve that in the `deps` file.